frontend/src/v4/routes/viewerGui/components/issues/components/issueDetails/issueDetails.component.tsx

```diff
@@ -34,6 +34,7 @@ import { IssueDetailsForm } from './issueDetailsForm.component';
 interface IProps {
 	viewer: any;
 	jobs: any[];
+	filteredIssues: any[];
```

Comment:
Let's wait for #4563 to be merged into staging because in that PR we get the filters to persist throughout the session.